src/PhpPact/Consumer/Matcher/Generators/RandomBoolean.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace PhpPact\Consumer\Matcher\Generators;
+
+use PhpPact\Consumer\Matcher\Model\GeneratorInterface;
+
+/**
+ * Generates a random boolean value
+ */
+class RandomBoolean implements GeneratorInterface
+{
+    /**
+     * @return array<string, string>
+     */
```

Comment:
thought after looking at several of these, is there not a common format that these fit into?

key, value

Reply:
I'm not sure I understand this question.

Are you suggesting that we can replace `@return array<string, xxx>` with `@return array<string, mixed>` so it's consistent between classes?

Reply:
I am suggesting making a data-transfer (readonly public) class to store named arguments, rather than returning an array.

Reply:
I can't define a DTO class like this:

```php
class CustomDto
{
    public function __construct(public string $pact:generator:type) {}
}
```

There is a syntax error because of the colon `:`

Reply:
I can prevent returning array by doing this:

```php
public function jsonSerialize(): object
{
        return (object) [
              'pact:generator:type' => 'RandomBoolean',
        ];
}
```

It's a bit hacky, but PHPStan will not complain, and test case is still passed. Then I can remove the `@return` annotation completely.

Is it fine to do this?

Reply:
I created a separate PR for it https://github.com/pact-foundation/pact-php/pull/389

Reply:
Another attempt https://github.com/pact-foundation/pact-php/pull/392